app/common/actions/tabActions.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const dispatchAction = require('../../common/dispatcher/dispatchAction')
+
+const tabActions = {
+  // TODO(bridiver) - this is an action anti-pattern and the code that uses it should be refactored
+  // to be declarative
+  reload: function tabActionsReload (tabId, ignoreCache) {
+    dispatchAction(tabActions.reload.name, {
```

Comment:
yea, I had actually commented on this in a previous version of the PR and said I wasn't super happy with it, but trying to find some way to avoid the constants file